src/backend/routes/views/clans/transfer.js

```diff
@@ -0,0 +1,32 @@
+const { JavaApiError } = require('../../../services/ApiErrors')
+
```

Comment:
Might want to add a validation back to this. We can assume that the single necessary param is here based on the view so might not be a real concern